templates/user/webhooks.html

```diff
@@ -58,35 +59,29 @@
-          <svg class="mr-2" id="icon" viewBox="0 0 32 32" style="width: 1.5rem; height: 1.5rem; fill: var(--font-color);" xmlns="http://www.w3.org/2000/svg">
-            <!-- credit https://www.iconfinder.com/carbon-design -->
-            <defs>
-              <style>
-                .webhook-cls-1 {
-                  fill: none;
-                }
-              </style>
-            </defs>
-            <path d="M24,26a3,3,0,1,0-2.8164-4H13v1a5,5,0,1,1-5-5V16a7,7,0,1,0,6.9287,8h6.2549A2.9914,2.9914,0,0,0,24,26Z" />
-            <path d="M24,16a7.024,7.024,0,0,0-2.57.4873l-3.1656-5.5395a3.0469,3.0469,0,1,0-1.7326.9985l4.1189,7.2085.8686-.4976a5.0006,5.0006,0,1,1-1.851,6.8418L17.937,26.501A7.0005,7.0005,0,1,0,24,16Z" />
-            <path d="M8.532,20.0537a3.03,3.03,0,1,0,1.7326.9985C11.74,18.47,13.86,14.7607,13.89,14.708l.4976-.8682-.8677-.497a5,5,0,1,1,6.812-1.8438l1.7315,1.002a7.0008,7.0008,0,1,0-10.3462,2.0356c-.457.7427-1.1021,1.8716-2.0737,3.5728Z" />
-            <rect class="webhook-cls-1" data-name="&lt;Transparent Rectangle&gt;" id="_Transparent_Rectangle_" />
-          </svg>
-          Webhooks
-        </h1>
-        <!-- <nav aria-label="breadcrumb">
-                    <ol class="breadcrumb font-size-1 mb-0" style="padding:0; background-color:transparent;">
-                        <li class="breadcrumb-item"><a href="/" title="Home">Home</a></li>
-                        <li class="breadcrumb-item active" aria-current="page">
-                            Webhooks 
-                        </li>
-                    </ol>
-                </nav> -->
+      <div class="d-md-flex justify-content-md-between my-d">
+        <div class="page-title">
+          <h4 class="h4 d-flex justify-content-center align-items-center">
+            <svg class="mr-2" id="icon" viewBox="0 0 32 32" style="width: 1.5rem; height: 1.5rem; fill: var(--font-color);" xmlns="http://www.w3.org/2000/svg">
+              <!-- credit https://www.iconfinder.com/carbon-design -->
+              <defs>
+                <style>
+                  .webhook-cls-1 {
+                    fill: none;
+                  }
+                </style>
+              </defs>
+              <path d="M24,26a3,3,0,1,0-2.8164-4H13v1a5,5,0,1,1-5-5V16a7,7,0,1,0,6.9287,8h6.2549A2.9914,2.9914,0,0,0,24,26Z" />
+              <path d="M24,16a7.024,7.024,0,0,0-2.57.4873l-3.1656-5.5395a3.0469,3.0469,0,1,0-1.7326.9985l4.1189,7.2085.8686-.4976a5.0006,5.0006,0,1,1-1.851,6.8418L17.937,26.501A7.0005,7.0005,0,1,0,24,16Z" />
+              <path d="M8.532,20.0537a3.03,3.03,0,1,0,1.7326.9985C11.74,18.47,13.86,14.7607,13.89,14.708l.4976-.8682-.8677-.497a5,5,0,1,1,6.812-1.8438l1.7315,1.002a7.0008,7.0008,0,1,0-10.3462,2.0356c-.457.7427-1.1021,1.8716-2.0737,3.5728Z" />
+              <rect class="webhook-cls-1" data-name="&lt;Transparent Rectangle&gt;" id="_Transparent_Rectangle_" />
+            </svg>
+            Webhooks
```

Comment:
Spacing differs from other headings

Reply:
tried to make it more compliant ... problem is the big button on the right that